docs/pages/roadmap.js

```diff
@@ -0,0 +1,676 @@
+    'link': '',
+  },
+  {
+    'task': 'Button component',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Android/iOS',
+    'description': 'Development of the Button component for Android and iOS.',
+    'link': '',
+  },
+  {
+    'task': 'IconButton component',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Android/iOS',
+    'description': 'Development of the IconButton component for Android and iOS.',
+    'link': '',
+  },
+  {
+    'task': 'Icon component',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Android',
+    'description': 'Development of the Icon component for Android.',
+    'link': '',
+  },
+  {
+    'task': 'Avatar component',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Android',
+    'description': 'Development of the Avatar component for Android.',
+    'link': '',
+  },
+  {
+    'task': 'RadioGroup component',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description':
+      'Create new component to create accessible Radio button groups in forms, deprecate singular RadioButton.',
+    'link': '',
+  },
+  {
+    'task': 'Masonry component enhancements',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description': 'Develop general technical improvements to Masonry.',
+    'link': '',
+  },
+  {
+    'task': 'Device type hooks',
+    'deadline': '',
+    'status': 'inProgress',
+    'platform': 'Web',
+    'description': 'Develop functionality to unlock device-specific UI in Gestalt.',
+    'link': '',
+  },
+  {
+    'task': 'Context logging hooks',
+    'deadline': '',
+    'status': 'inProgress',
+    'platform': 'Web',
+    'description':
+      'Develop functionality to allow for generic logic to be added on component interaction.',
+    'link': '',
+  },
+  {
+    'task': 'Border color and radius design tokens',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description': 'Addition of tokens to support border radius and border color styling.',
+    'link': '',
+  },
+  {
+    'task': 'Color opacity design tokens',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description': 'Addition of tokens to support opacity styling.',
+    'link': '',
+  },
+  {
+    'task': 'Elevation design tokens',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description':
+      'Addition of tokens to support the styling of elevated elements within a layout.',
+    'link': '',
+  },
+  {
+    'task': 'Gestalt docs IA update',
+    'deadline': 'Quarter 1',
+    'status': 'inProgress',
+    'platform': '',
+    'description':
+      'Update of the Gestalt docs site&rsquo;s information architecture to support pattern guidelines and mobile component documentation.',
+    'link': '',
+  },
+  {
+    'task': 'Component visual search',
```

Comment:
Maybe "Visual component overview" based on this morning?

Reply:
+1

Reply:
Fixed!